fixtures/flows/brief/atm.js

```diff
@@ -1,136 +1,161 @@
-export const create = async (params) => {
-    console.log('Starting to create atm brief');
-    await createBrief();
-    await fillWhoCanRespond();
-    await fillAbout(params.title, params.locations);
-    await fillResponseFormats();
-    await fillObjectives();
-    await fillTimeframes();
-    let criterias = await fillResponseCriteria(params.numberOfCriterias ? params.numberOfCriterias : 2);
-    await fillClosingDate();
-    await publishBrief();
-    return {
-        criterias
-    };
+import * as util from '../utils'
+
+/* eslint-disable no-await-in-loop */
```

Comment:
is this still needed? if so can it be put closer to where the line is with `eslint-disable-next-line`